module/Activity/view/email/activity.phtml

```diff
@@ -0,0 +1,37 @@
+<?php $lang = 'en'; ?>
+<?= $this->translate('Hello,')?><br /><br />
```

Comment:
Won't this result in emails being sent in different languages depending on what language the creator of the activity is using?